FirebaseSessions/Sources/EventFireLogger.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+@_implementationOnly import GoogleDataTransport
+
+protocol EventFireLoggerProtocol {
+  func logEvent(event: SessionStartEvent, completion: @escaping (Result<Void, Error>) -> Void)
+}
+
+class EventFireLogger: EventFireLoggerProtocol {
```

Comment:
A design suggestion: Can we set this class in such a way that we bring in an option (dispatch mode) to just dump the event to the console rather than dispatch to Firelog? This would be helpful for our local testing where we don't have to wait for long to see the events to reach Firelog to ascertain that the events are as expected. FWIW: Default is Dispatch to Firelog.

Reply:
Added it to Taskflow